mysql-test/suite/versioning/t/online.combinations

```diff
@@ -0,0 +1,2 @@
+[inplace]
+[instant]
```

Comment:
I guess you confuses online and instant.

`INPLACE` is every alter operation which InnoDB is capable to do by itself.
`NOCOPY` is a faster subset of all `INPLACE` operations.
`INSTANT` is an even faster subset on `NOCOPY` operations. These operations just changes metadata, like renames columns.

Online alter is an asynchronous operations. It creates a new index, copies visible by current transaction data from the old one, then it applies operations from undo log (which was asyncronously performed by other connections) and replaces an old index with the new one. `LOCK=NONE` means online alter, not instant.

Reply:
True. But I suppose that non-versioned operations should work as **online** on system versioned tables as well. F.ex. there is no difference for `DROP COLUMN` or `ADD INDEX` whether the table is versioned or not. `WITHOUT SYSTEM VERSIONING` should not be affected altogether since it is instant operation. So I propose to disable only `ADD SYSTEM VERSIONING` and `DROP SYSTEM VERSIONING`. The appended commit contains possible solution.

Reply:
> But I suppose that non-versioned operations should work as online on system versioned tables as well.

Yes, I've tried some other alter operations the do work fine online. Actually I don't know any alter types that don't work with system-versioned table. So having a fine-grained check is good.

> WITHOUT SYSTEM VERSIONING should not be affected altogether since it is instant operation.

Why `instant`? It removes historical rows. Modyfying data is not `instant`. Only metadata change is.

> So I propose to disable only ADD SYSTEM VERSIONING and DROP SYSTEM VERSIONING.

I think even `ADD SYSTEM VERSIONING` can be allowed.

Reply:
`WITHOUT SYSTEM VERSIONING` (not `DROP SYSTEM VERSIONING`) is about column, it just adds property to a field.

Reply:
Oh yes. Sorry for my inattention. It's unaffected indeed.